smart-contracts/Artifacts.md

```diff
@@ -0,0 +1,22 @@
+# This is a document to describe how we save our smart contract deployments to a central location in git as a team.
+
+## Problem
+
+Truffle has a directory called build that gets generated whenever contracts are compiled. This directory also keeps track of the networks and the addresses contracts are deployed to on various networks. Unfortunately, truffle only allows you to save 1 smart contract address per network. This presents a problem as we still need to track the addresses of the same contract that has deployed to a single testnet multiple times for different smart contract instances.
+
+## Solution
+
+In order to remedy this problem, we have created a script called ```saveContracts.js``` located inside of the scripts folder. Before you do a deployment to the network of your choosing, delete the build folder, then run your migrations. After the contracts are successfully deployed, then from the smart-contracts directory, run the saveContracts.js script. Be sure to pass a name as an environment variable when you run the script of the new folder where these artififacts and addresses will be deployed. 
```

Comment:
Do you also need to delete the openzeppelin folder?

Reply:
this PR deleted the openzeppelin folder so that shouldn't be checked in at the root of the smart contracts folder

Reply:
openzeppelin will be included later in another PR in git since only with this folder upgradability works.

Reply:
Please take a look at the script at the end. It now copies the openzeppelin directory into the folder you are creating